config/default/kustomization.yaml

```diff
@@ -25,11 +25,6 @@ bases:
 # - ../prometheus
 
 patchesStrategicMerge:
-# Protect the /metrics endpoint by putting it behind auth.
-# If you want your controller-manager to expose the /metrics
-# endpoint w/o any authn/z, please comment the following line.
-- manager_auth_proxy_patch.yaml
```

Comment:
Hi @ChunyiLyu 

You removed the protection now it is unprotected see; https://github.com/syntasso/kratix/blob/41abb4c8fe4ba11a6c4e6675cde4082f0f6d0b3a/charts/kratix/templates/distribution.yaml#L429

Please, you must ensure that it is protected. 
**Check out the FAQ section: ["How can I manually change my project to switch to Controller-Runtime's built-in auth protection?"](https://github.com/kubernetes-sigs/kubebuilder/discussions/3907) for detailed instructions.**

Just update your scaffold referent in this area as described in the step-by-step tutorial.
Validate it and ensure that it is protected.

Reply:
Hi @camilamacedo86 thanks for looking into the changes. This file [kratix/charts/kratix/templates/distribution.yaml](https://github.com/syntasso/kratix/blob/41abb4c8fe4ba11a6c4e6675cde4082f0f6d0b3a/charts/kratix/templates/distribution.yaml#L429) is a placeholder that gets replaced entirely in our CI and *not* what we release. What you linked is not included the released version our helm chart.